api/tx_args.go

```diff
@@ -138,7 +143,8 @@ func (args *SendTxArgs) setDefaults(ctx context.Context, b Backend) error {
 		args.GasLimit = new(hexutil.Uint64)
 		*args.GasLimit = hexutil.Uint64(90000)
 	}
-	if args.Price == nil {
+	// For the transaction that do not use the gasPrice field, the default value of gasPrice is not set.
+	if args.Price == nil && *args.TypeInt != types.TxTypeEthereumDynamicFee {
```

Comment:
We can set `GasTipCap` and `GasFeeCap` for  TxTypeEthereumDynamicFee referring `func (args *EthTransactionArgs) setDefaults`.  How do think about it?

Reply:
Umm.. i think i can add default GasFeeCap and GasTipCap logic in `setDefaults`.